src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] arrayOfFigure = new Figure[6];
+        final int First_Half_Of_Array = 3;
```

Comment:
Magic numbers should be replaced with constants. Also, the naming convention for constants is to use UPPER_CASE with underscores. Consider renaming `First_Half_Of_Array` to `FIRST_HALF_OF_ARRAY`.